src/explainer_prep.R

```diff
@@ -0,0 +1,193 @@
+#' @param blue_label label for wet condition interpretation tip
+#' @param orange_label label for dry condition interpretation tip
+cowplot_national_explainer <- function(explainer_label, file_png, national_plot_png,
+                                       width, height, font_legend, text_color, blue_label, orange_label){
+
+  plot_margin <- 0.025
+  # background
+  canvas <- grid::rectGrob(
+    x = 0, y = 0, 
+    width = 16, height = 9,
+    gp = grid::gpar(fill = color_bknd, alpha = 1, col = color_bknd)
+  )
+  
+  #arrows
+  (normal_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = text_color, linewidth = 0.2))
+  
+  (low_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = "#A84E0B", linewidth = 0.2))
+  
+  (high_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = "#002D5E", linewidth = 0.2))
+  
+  og_plot_png <- magick::image_read(national_plot_png)
+  # compose final plot
+  ggdraw(ylim = c(0,1), 
+         xlim = c(0,1)) +
+    # a white background
+    draw_grob(canvas,
+              x = 0, y = 1,
+              height = 1, width = 1,
+              hjust = 0, vjust = 1) +
+    draw_image(og_plot_png, 
+               x = 0, y = 0, 
+               width = 1, 
+               hjust = 0, vjust = 0, 
+               halign = 0, valign = 0)+
+    draw_label(explainer_label, 
+               x = 0.5, y = 0.96, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#000000")+ #text_color) +
+    draw_label(blue_label, 
+               x = 0.5, y = 0.91, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#002D5E")+
+    draw_label(orange_label, 
+               x = 0.5, y = 0.88, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#A84E0B")+
+    draw_label("Low\nStreamflow",
+               x = 0.84, y = 0.355, 
+               size = 5.5, 
+               hjust = 0.5, vjust = 1,
+               fontfamily = font_legend,
+               color = "#A84E0B") +
+    draw_label("High\nStreamflow",
```

Comment:
can also make these fxn parameters in `cowplot_national_explainer` to easily edit down the line if needed